controls/radtreeview/features/ui-virtualization.md

```diff
@@ -71,54 +62,51 @@ Now when you try to expand the "Teams" node, then only those elements that might
 
 >Where the __telerikTreeView__ alias points to the __Telerik.Windows.Controls.TreeView__ namespace in the __Telerik.Windows.Controls.Navigation__ assembly.
 
-## Setting the VirtualizationMode
+## Setting the TreeVirtualizationMode
+
+When you want to specify the method the __TreeViewPanel__ uses to manage virtualizing its child items, then you should set the __TreeVirtualizationMode__ property of the __TreeViewPanel__.
 
-When you want to specify the method the __TreeViewPanel__ uses to manage virtualizing its child items, then you should set the __VirtualizationMode__ property of the __TreeViewPanel__.
+The __TreeVirtualizationMode__ property is a __VirtualizationMode__ enumeration which may accept the following values:
 
-The __VirtualizationMode__ property is a __VirtualizationMode__ enumeration which may accept the following values:
+> In some version of Visual Studio MS XAML parsel could throw a 'Ambiguous match found.' error in the output when the __VirtualizationMode__ property is used. This error comes from the fact that __TreeViewPanel__ inherits from the MS VirtualizingPanel which in .Net4.5 exposes that same property and which is hiding an inheritance property. The XAML parser is designed to work in that way so we have introduced another attached property which is doing the same thing. We suggest to use the telerik:TreeViewPanel.TreeVirtualizationMode property instead.

```

Comment:
* MS XAML __Parser__
* that same property which is hiding an ineritance property (without and)